include/s3select_oper.h

```diff
@@ -281,26 +271,26 @@ class s3select_reserved_word
   enum class reserve_word_en_t
   {
     NA,
-    S3S_NULL,//TODO check AWS defintions for reserve words, its a long list , what about functions-names? 
+    S3S_NULL,//TODO check AWS definitions for reserve words, its a long list , what about functions-names?
     S3S_NAN 
   } ;
 
   using reserved_words = std::map<std::string,reserve_word_en_t>;
 
-  const reserved_words m_reserved_words=
+  inline static const reserved_words m_reserved_words=
   {
     {"null",reserve_word_en_t::S3S_NULL},{"NULL",reserve_word_en_t::S3S_NULL},
     {"nan",reserve_word_en_t::S3S_NAN},{"NaN",reserve_word_en_t::S3S_NAN}
   };
 
-  bool is_reserved_word(std::string & token)
+  static bool is_reserved_word(std::string & token)
```

Comment:
for what purpose is the static qualifier ?

Reply:
to indicate that this is actually a standalone function that does not need an instance of the object. 
in theory, you could have taken the function declaration outside if the class.
if this is likely to change (e.g. in the future this function will look into data members of the class) than keep it without the "static" keyword.
if not, than either add "static" or take out of the class.

Reply:
i'm aware for that role of static.
the _is_reserved_word_ is not stand for itself (should be a member)